improver/calibration/ensemble_calibration.py

```diff
@@ -562,114 +557,201 @@ def __repr__(self):
             self.max_iterations,
         )
 
-    def create_coefficients_cube(self, optimised_coeffs, historic_forecast):
-        """Create a cube for storing the coefficients computed using EMOS.
+    def _multiple_beta_coefficients(self, optimised_coeffs, historic_forecast):
```

Comment:
How about `_restructure_coefficients` (as this is what the docstring says it does).

Reply:
This method has now actually been removed.